templates/_helpers.tpl

```diff
@@ -32,3 +32,28 @@ stackstorm
 {{- define "hyphenPrefix" -}}
 {{ if . }}-{{ . }}{{end}}
 {{- end -}}
+
+# Allow calling helpers from nested sub-chart
+# https://stackoverflow.com/a/52024583/4533625
+# https://github.com/helm/helm/issues/4535#issuecomment-477778391
+# Usage: "{{ include "nested" (list . "mongodb-ha" "mongodb-replicaset.fullname") }}"
+{{- define "nested" }}
+{{- $dot := index . 0 }}
+{{- $subchart := index . 1 | splitList "." }}
+{{- $template := index . 2 }}
+{{- $values := $dot.Values }}
+{{- range $subchart }}
+{{- $values = index $values . }}
+{{- end }}
+{{- include $template (dict "Chart" (dict "Name" (last $subchart)) "Values" $values "Release" $dot.Release "Capabilities" $dot.Capabilities) }}
```

Comment:
All this trickery should be eventually covered by respective Unit Tests https://github.com/StackStorm/stackstorm-ha/issues/28